draft-chen-cfrg-vdaf-pine.md

```diff
@@ -307,8 +307,33 @@ We define a function `PineValid.encode_gradient(self, measurement: list[float])
 
 ### Running the Wraparound Tests {#run-wr-check}
 
-> XXX Give an overview of how the wraparound test results are generated from
-> the XOF seed.
+Given the encoded gradient from {{encode-gradient}} and the XOF to generate the
```

Comment:
Harmonize with #65 once you have a PR up for that.

Reply:
Done.